src/components/campaigns/grid/EditForm.tsx

```diff
@@ -229,6 +232,27 @@ export default function EditForm({ campaign }: { campaign: AdminSingleCampaignRe
               autoComplete="title"
             />
           </Grid>
+          <Grid item xs={12}>
+            <FormTextField
+              type="text"
+              label="campaigns:campaign.slug.name"
+              name="slug"
```

Comment:
it will be great to have the ability to specify the slug manually in the CreateForm too - so could you pls add the field as editable there too. May be without the onchange warning because the campaign is still not created.

Note: If a user tries to create a slug that already exists, the unique-constraint on the backend will return error and the creation will not succeed, so we are safe here.
However, would be good to make the warning and error message more descriptive to show the reason. If not easy, let's split it in separate PR.

Reply:
- Agree about the table with the campaign slugs on the BE. Will address it in a separate PR.
- Will add the slug field to the create form as well
- I think the unique constraint is actually not handled anywhere on the FE - it is the same on user registration. If you try to register an existing email , a general message appears and it's hard to tell why you could not register. It's definitely a must to display it from UX perspective. I'll fix that and if I find it too complex - will split it in a separate PR.

Reply:
## UPDATE

- Added the Slug Field to the Create Form as well - the behavior is as follow
    - Field is optional
    - It contains a placeholder value instructing the user that the default value will be the one for `campaign title`, unless he/she decides to set it to a custom one
    - If left empty, the` campaign slug` is the `campaign title`, else it's the value from the field

* Regarding displaying a message for when an unique constraint is violated, I'll do this in a separate PR - to keep things organized and focused